larray/inout/csv.py

```diff
@@ -31,12 +31,13 @@ def read_csv(filepath_or_buffer, nb_axes=None, index_col=None, sep=',', headerse
     Notes
     -----
     csv file format:
-    arr,ages,sex,nat\time,1991,1992,1993
-    A1,BI,H,BE,1,0,0
-    A1,BI,H,FO,2,0,0
-    A1,BI,F,BE,0,0,1
-    A1,BI,F,FO,0,0,0
-    A1,A0,H,BE,0,0,0
+    geo,gender\\time,2015,2016,2017
+    BE,M,5524068.0,5569264.0,5589272.0
```

Comment:
wouldn't using longer labels make things more readable? (BE -> Belgium, M -> Male, ...)?